Sources/Sentry/SentryHttpTransport.m

```diff
@@ -123,14 +117,9 @@ - (void)setupQueueing
         // If a response from the server is received, regardless of whether the
         // request completes successfully or fails, the response parameter
         // contains that information.
-        if (response == nil) {
-            // In case response is nil, we want to queue the event locally since
-            // this indicates no internet connection
-            return YES;
-        }
         // In all other cases we don't want to retry sending it and just discard
         // the event
-        return NO;
+        return response == nil;
```

Comment:
M: We are losing this comment:

>             // In case response is nil, we want to queue the event locally since	
>             // this indicates no internet connection

its useful IMO

Reply:
no connection (try again) != server error (delete it and keep going), so the comment describes its behaviour

Reply:
Ops, good catch thank you 💯